### DISCUSSION TITLE
the button in sm: w-full

### DISCUSSION THREAD

**Discussion by GajdosGabriel** [tailwindlabs/tailwindcss#1961] [Help]
Hi,

how do make the button in sm: w-full and md and lg
normal size.
Thx a lot.

Gabriel

**Comment by devcircus** [comment]
```<button class="w-full md:w-12">```

And w-full depends on the width of the parent.

**Reply by estevanmaito** [reply]
Instead of using a fixed width, you could use `md:w-auto`, so it gets it's default width.

**Reply by GajdosGabriel** [reply]
Yes, it's good solution.
Thx a lot
